src/containers/main/UpgradeContainer/index.tsx

```diff
@@ -280,7 +280,7 @@ export const UpgradeContainer: FC<IProps> = ({ address, balance }) => {
                 .times(usdPrice);
 
               return (
-                <tr>
+                <tr key={token.coin}>
```

Comment:
`token.coin` can be the `key` prop because it is **unique** and also **stable**.